client/Main.js

```diff
@@ -1 +1,2 @@
-import '../imports/ui/Body.js';
\ No newline at end of file
+BlazeLayout.setRoot('body');
```

Comment:
Can you tell me what this does?

Reply:
This allows us to set the root of the Blaze renderer to the <body> tag, instead of its own special one. This is only so it doesn't look like
```
<body>
</body>
<div>
</div>